src/data/buildings.ts

```diff
@@ -1,8 +1,61 @@
 import { REQUIRE_ALL_CARD_PROPERTIES } from "./config";
-import { getProperties } from "./helpers";
+import {
+  active,
+  constantModifiers,
+  effectBase,
+  getProperties,
+  queryParam,
+} from "./helpers";
 
 export const getBuildingProperties = getProperties(id => {
+  let aid = 1;
   switch (id) {
+  case "Verdant Tree":
+    return {
+      attributes: {
+        HEALING: 1,
+      },
+      activatedAbilities: [
+        active(
+          id,
+          aid++,
+          [
+            {
+              type: "EXHAUST_THIS",
+            },
+          ],
+          ($, I) => {
+            const player = $.getPlayer(I.controller);
+
+            if (player != null) {
+              return [
+                {
+                  ...effectBase(id, I, "MODIFY"),
+                  target: queryParam("INSTANCE", {
```

Comment:
nit: I think this might have limitations long-term? I think we may need the effect to exist on the player or sth so that tech buildings that haven't been purchased yet would still instantly build... maybe it's at least worth a comment or issue to document this in case we can come back and improve it later?

Reply:
I'll leave a note. I must have anticipated that this approach would work even for buildings that have not been built yet.